Sources/AardvarkTests/ViewHierarchyAttachmentGeneratorTests.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  Copyright © 2021 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+
+@testable import Aardvark
+
+final class ViewHierarchyAttachmentGeneratorTests: XCTestCase {
+
+    func testAttachmentMetadata() {
+        let attachment = ViewHierarchyAttachmentGenerator.generateAttachment(for: [])
+
+        XCTAssertEqual(attachment.fileName, "view_hierarchy.txt")
+        XCTAssertEqual(attachment.dataMIMEType, "text/plain")
+    }
```

Comment:
It's hard to look at this method name and understand what will be tested. I think it'd be best to bring these two `XCTAssert` calls into two separate tests. I'd write this as:

```suggestion
    func test_generateAttachment_hasExpectedFilename() {
        let attachment = ViewHierarchyAttachmentGenerator.generateAttachment(for: [])

        XCTAssertEqual(attachment.fileName, "view_hierarchy.txt")
    }

    func test_generateAttachment_hasExpectedMIMEType() {
        let attachment = ViewHierarchyAttachmentGenerator.generateAttachment(for: [])

        XCTAssertEqual(attachment.dataMIMEType, "text/plain
    }
```

Reply:
Addressed in f3b8f0a.